docs/developers.md

```diff
@@ -11,9 +9,16 @@ There are 3 distributions in this project:
 Both `git-keeper-client` and `git-keeper-server` depend on `git-keeper-core`.
 
 When developing, all 3 distributions should be installed in development mode
-by running `python setup.py develop` in the top level directory of all 3
+by running `python3 setup.py develop` in the top level directory of all 3
 distributions, starting with `git-keeper-core`.
```

Comment:
I think we should include an explanation of what  `develop` does - or at least include a link to where this is described:

http://pythonhosted.org/setuptools/setuptools.html#develop-deploy-the-project-source-in-development-mode